nx/include/switch/services/set.h

```diff
@@ -262,6 +262,128 @@ typedef struct {
     u32 extra_color;      ///< Extra Color.
 } SetSysHomeMenuScheme;
 
+typedef struct {
+    u8 offset[0x6];
+} SetCalAccelerometerOffset;
+
+typedef struct {
+    u8 scale[0x6];
+} SetCalAccelerometerScale;
+
+typedef struct {
+    u8 cert[0x70];
+} SetCalAmiiboEcdsaCertificate;
+
+typedef struct {
+    u8 cert[0x20];
+} SetCalAmiiboEcqvBlsCertificate;
+
+typedef struct {
+    u8 key[0x44];
+} SetCalAmiiboEcqvBlsKey;
+
+typedef struct {
+    u8 cert[0x90];
+} SetCalAmiiboEcqvBlsRootCertificate;
+
+typedef struct {
+    u8 cert[0x14];
+} SetCalAmiiboEcqvCertificate;
+
+typedef struct {
+    u8 key[0x54];
+} SetCalAmiiboKey;
+
+typedef struct {
+    u8 calibration[0x9];
+} SetCalAnalogStickFactoryCalibration;
+
+typedef struct {
+    u8 parameter[0x12];
+} SetCalAnalogStickModelParameter;
+
+typedef struct {
+    u8 battery_lot[0x18];
+} SetCalBatteryLot;
+
+typedef struct {
+    u8 bd_addr[0x6];
+} SetCalBdAddress;
+
+typedef struct {
+    u8 cfgId1[0x1E];
+} SetCalConfigurationId1;
+
+typedef struct {
+    u8 offset[0x6];
+} SetCalConsoleSixAxisSensorHorizontalOffset;
+
+typedef struct {
+    u32 code[0x4];
+} SetCalCountryCode;
+
+typedef struct {
+    u32 offset;                     ///< Relative to current position.
```

Comment:
Align these comments with each other (similar to elsewhere).

Reply:
I have aligned them, hope it looks okay.